mandatory/4-tax.js

```diff
@@ -17,7 +21,13 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price){
```

Comment:
Very clean code, well done! There are various options to the functions you are coding to make it shorter. Making use of the correct methods in your code, are best practise, Well done there!